projects/ng-components-docs/src/app/examples/components/progressBar/progressBar.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit, OnDestroy } from "@angular/core";
+import { BarItem } from "lib/src/public_api";
```

Comment:
is it ok to import BarItem from public_api ? i believe we removed this public_api and instead you can import BarItem from the progress component

Reply:
The VS code auto import it seems. Yah will reference the component.